lib/vicadmin/validate.go

```diff
@@ -55,6 +54,9 @@ type Validator struct {
 	DockerPort       string
 	VCHStatus        template.HTML
 	VCHIssues        template.HTML
+	VCHReachable     bool
+	VCHUnreachable   bool
```

Comment:
Fixed. Turns out the template doesn't like html comments between `{{if}}` and `{{else}}`.